extensions/units.lua

```diff
@@ -10,6 +10,7 @@ HodorReflexes.units = {}
 HR_EVENT_UNITS_UPDATED = "UnitsUpdated"
 
 local HR = HodorReflexes
+local M = HR.modules.share
```

Comment:
share module is not needed here.
M describes only the current module of the file and not allways "share"